lib/indexer/document_preparer.rb

```diff
@@ -27,7 +38,7 @@ def prepare_popularity_field(doc_hash, popularities)
     end
 
     def prepare_tags_field(doc_hash)
-      Indexer::TagLookup.new.prepare_tags(doc_hash)
```

Comment:
What's the value of changing the interface? How I see it, if we have to add more args to the method, we'd have to change code in four places instead of two.

If the empty initializer os really bothering you I'd prefer changing it into `TagLookup.new(doc_hash).doc_hash_with_tags`